kakao/src/main/kotlin/com/agoda/kakao/list/AbsListViewAdapterAssertions.kt

```diff
@@ -0,0 +1,21 @@
+@file:Suppress("unused")
+
+package com.agoda.kakao.list
+
+import androidx.test.espresso.assertion.ViewAssertions
+import com.agoda.kakao.common.assertions.AdapterAssertions
+import com.agoda.kakao.common.matchers.KAbsListViewAdapterSizeMatcher
+
+/**
+ * Provides assertions for AbsListView adapter
+ */
+interface AbsListViewAdapterAssertions : AdapterAssertions {
+    /**
+     * Check size of AbsListView
+     *
+     * @param size expected child count size in AbsListView
+     */
+    fun hasSize(size: Int) {
+        view.check(ViewAssertions.matches(KAbsListViewAdapterSizeMatcher(size)))
+    }
+}
```

Comment:
Please don't forget to add newline at the end of the file

Reply:
Done